experimental/dds/tree/src/EditLog.ts

```diff
@@ -225,10 +252,10 @@ export class EditLog<TChange = unknown> extends TypedEventEmitter<IEditLogEvents
 	public readonly editsPerChunk: number;
 
 	/**
-	 * @returns The index of the earliest edit available through `getEditInSessionAtIndex`.
+	 * @returns The index of the earliest edit that can be accessed through `tryGetEditInSessionAtIndex`. This edit can have already been evicted.
```

Comment:
I think rather than "this can have already been evicted", it would be clearer to say that the edit may not be stored in memory. 

Additionally, if I'm understanding correctly, we now have the distinction between the set of all edits ever, the set created during the current session, and the set that is stored in memory. Why is the second needed?

Reply:
I keeped the second behavior for not introducing a breaking change.